moto/redshiftdata/models.py

```diff
@@ -0,0 +1,192 @@
+import re
+import uuid
+from datetime import datetime
+import random
+
+from moto.core import BaseBackend
+from moto.core.utils import BackendDict, iso_8601_datetime_without_milliseconds
+from moto.redshiftdata.exceptions import ValidationException, ResourceNotFoundException
+
+
+class Statement:
+    def __init__(
+        self,
+        cluster_identifier,
+        database,
+        db_user,
+        query_parameters,
+        query_string,
+        secret_arn,
+    ):
+        now = iso_8601_datetime_without_milliseconds(datetime.now())
+
+        self.Id = str(uuid.uuid4())
```

Comment:
Variables and attribute-names should be `snake_case` everywhere, to be consistent

Reply:
Sure. Done